cypress/support/commands.tsx

```diff
@@ -104,7 +104,7 @@ Cypress.Commands.add("mount", function (children, options) {
   };
 
   return cypressMount(
-    <ToolkitProvider density={this.density} theme={this.theme}>
+    <ToolkitProvider density={this.density} mode={this.theme}>
```

Comment:
```suggestion
    <ToolkitProvider density={this.density} mode={this.mode}>
```
See line 65~ That will need to be updated too